Source/WebKit/NetworkProcess/NetworkResourceLoadParameters.cpp

```diff
@@ -31,406 +31,111 @@
-            if (auto* fileData = std::get_if<FormDataElement::EncodedFileData>(&element.data)) {
-                const String& path = fileData->filename;
-                if (auto handle = SandboxExtension::createHandle(path, SandboxExtension::Type::ReadOnly))
-                    requestBodySandboxExtensions.append(WTFMove(*handle));
-            }
-        }
-        encoder << requestBodySandboxExtensions;
-    }
-
-    if (request.url().isLocalFile()) {
-        SandboxExtension::Handle requestSandboxExtension;
-#if HAVE(AUDIT_TOKEN)
-        if (networkProcessAuditToken) {
-            if (auto handle = SandboxExtension::createHandleForReadByAuditToken(request.url().fileSystemPath(), *networkProcessAuditToken))
-                requestSandboxExtension = WTFMove(*handle);
-        } else
+NetworkResourceLoadParameters::NetworkResourceLoadParameters(WebCore::ResourceLoaderIdentifier identifier, WebPageProxyIdentifier webPageProxyID, WebCore::PageIdentifier webPageID, WebCore::FrameIdentifier webFrameID, WTF::ProcessID parentPID, WebCore::ResourceRequest resourceRequest, RefPtr<WebCore::FormData> httpBody, std::optional<Vector<SandboxExtension::Handle>> sandboxExtensionIfHttpBody, std::optional<SandboxExtension::Handle> sandboxExtensionIflocalFile, WebCore::ContentSniffingPolicy contentSniffingPolicy, WebCore::ContentEncodingSniffingPolicy contentEncodingSniffingPolicy, WebCore::StoredCredentialsPolicy storedCredentialsPolicy, WebCore::ClientCredentialPolicy clientCredentialPolicy, PreconnectOnly shouldPreconnectOnly, bool shouldClearReferrerOnHTTPSToHTTPRedirect, bool needsCertificateInfo, bool isMainFrameNavigation, bool isMainResourceNavigationForAnyFrame, WebCore::ShouldRelaxThirdPartyCookieBlocking shouldRelaxThirdPartyCookieBlocking, Seconds maximumBufferingTime, RefPtr<WebCore::SecurityOrigin> sourceOrigin, RefPtr<WebCore::SecurityOrigin> topOrigin, WebCore::FetchOptions options, std::optional<WebCore::ContentSecurityPolicyResponseHeaders> cspResponseHeaders, URL parentFrameURL, URL frameURL, WebCore::CrossOriginEmbedderPolicy parentCrossOriginEmbedderPolicy, WebCore::CrossOriginEmbedderPolicy crossOriginEmbedderPolicy, WebCore::HTTPHeaderMap originalRequestHeaders, bool shouldRestrictHTTPResponseAccess, WebCore::PreflightPolicy preflightPolicy, bool shouldEnableCrossOriginResourcePolicy, Vector<RefPtr<WebCore::SecurityOrigin>> frameAncestorOrigins, bool pageHasResourceLoadClient, std::optional<WebCore::FrameIdentifier> parentFrameID, bool crossOriginAccessControlCheckEnabled, OptionSet<WebCore::NetworkConnectionIntegrity> networkConnectionIntegrityPolicy, bool allowPrivacyProxy, URL documentURL, bool isCrossOriginOpenerPolicyEnabled, bool isClearSiteDataHeaderEnabled, bool isDisplayingInitialEmptyDocument, WebCore::SandboxFlags effectiveSandboxFlags, URL openerURL, WebCore::CrossOriginOpenerPolicy sourceCrossOriginOpenerPolicy, uint64_t navigationID, std::optional<WebCore::NavigationRequester> navigationRequester,
```

Comment:
Can we remove all written-out constructors instead?  If not, can we put each parameter on its own line?  We usually don't, but for this many parameters it'll help with making git history more visible.